images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -149,6 +150,7 @@ $markdown += New-MDList -Style Unordered -NoNewLine -Lines @(
     "Hub CLI: ${hubVersion}",
     "GNU Wget ${wgetVersion}",
     "Subversion (SVN) ${svnVersion}",
+    "aws-iam-authenticator $awsIamAuthenticatorVersion",

```

Comment:
Let's move it under ths line
https://github.com/actions/virtual-environments/blob/55d0c3bed6d1c8e1480a85a2d4965add4940b9fc/images/macos/software-report/SoftwareReport.Generator.ps1#L204